.env

```diff
@@ -22,14 +22,17 @@ REACT_APP_NETWORK_URL_100=https://rpc.xdaichain.com
 REACT_APP_PORTIS_ID="c0e2bf01-4b08-4fd5-ac7b-8e26b58cd236"
 REACT_APP_FORTMATIC_KEY="pk_live_F937DF033A1666BF"
 
-# Domain
+# Domain regex (to detect environment)
 REACT_APP_DOMAIN_REGEX_LOCAL="^(:?localhost:\d{2,5}|(?:127|192)(?:\.[0-9]{1,3}){3})"
 REACT_APP_DOMAIN_REGEX_PR="^pr\d+--gpswapui\.review"
 REACT_APP_DOMAIN_REGEX_DEV="^cowswap\.dev"
 REACT_APP_DOMAIN_REGEX_STAGING="^cowswap\.staging"
 REACT_APP_DOMAIN_REGEX_PROD="^cowswap\.exchange$"
-REACT_APP_DOMAIN_REGEX_ENS="^cowswap\.eth$"
 REACT_APP_DOMAIN_REGEX_BARN="^barn\.cowswap\.exchange$"
+REACT_APP_DOMAIN_REGEX_ENS="(:?^cowswap\.eth$|ipfs)"
+
+# Path regex (to detect environment)
+REACT_APP_PATH_REGEX_ENS="/ipfs"
```

Comment:
What kind of URL's does this detect? Just trying to see if it has the chance to trigger false positives.

Reply:
Urls with `/ipfs` on them. 
I think should be fine as long as we don't include that in the path for any URL.

The other PR with the test has some examples of URL detections.

Reply:
Merging, happy to address in other PRs if u find any false positive examples